packages/odyssey-react-mui/src/Callout.tsx

```diff
@@ -29,7 +31,17 @@ export type CalloutProps = {
   /**
    * The contents of the Callout
    */
-  children: ReactNode;
+  children?: ReactNode;
+  /**
+   * If linkUrl is not undefined, this is the text of the link.
+   * If left blank, it defaults to "Learn more".
+   * Note that linkText does nothing if linkUrl is not defined
+   */
+  linkText?: string;
+  /**
+   * If defined, the Toast will include a link to the URL
+   */
+  linkUrl?: string;
```

Comment:
You should move these comments to the type changes you made below and delete these properties from here. I don't believe it will work if you leave these here and add the changes you did below with my previous suggestion.